src/components/grid/grid.ts

```diff
@@ -113,10 +113,6 @@ export class Grid extends Component<Props, SpreadsheetChildEnv> {
     useGridDrawing("canvas", this.env.model, () =>
       this.env.model.getters.getSheetViewDimensionWithHeaders()
     );
-    useEffect(
-      () => this.focus(),
-      () => [this.env.model.getters.getActiveSheetId()]
```

Comment:
removing this unfortunately does not work properly. most of the time it will work because a click on the bottom bar will bubble up to Spreadsheet component which will focus the grid but if the said click event is stopped at some point, then you are screwed.
Example of a broken behaviour now,
go to the bottom bar and duplicate the sheet
-> now the activeElement is body, grid no longer has the focus.

Might be worth adding a test for this

Reply:
the focus function already has an exception if we are selecting a figure, you could add one if the edition mode is not `inactive`for example

Reply:
Note that every other option in the bototm bar (except duplicated sheet) doesn't give the focus back to the grid in master.  Added it in https://www.odoo.com/web#id=3094579&cids=1&menu_id=4720&action=333&active_id=2328&model=project.task&view_type=form